source/style-guide/writing/write-for-accessibility.txt

```diff
@@ -95,12 +98,12 @@ heading. Make the hierarchy clear and headings meaningful.
   their browser and the page. It's easier to jump between pages and
   sections of a page if the headings and titles are unique.
 - Use a heading hierarchy and don't skip levels. For example, put an
-  ``h3`` only under an ``h2>``.
+  ``h3`` only under an ``h2``.
```

Comment:
**nit:** Should link to markup guide on heading types once created.

Reply:
[Noted](https://docs.google.com/document/d/1gwg44VP4_P-hDn-QhLEDOpon-ZnzfEFLxmIa4qy1kgQ/edit?usp=sharing).